packages/blade/src/components/Chip/Chip.tsx

```diff
@@ -158,9 +170,20 @@ const _Chip: React.ForwardRefRenderFunction<BladeElementRef, ChipProps> = (
         onKeyDown={handleKeyboardPressedIn}
         onKeyUp={handleKeyboardPressedOut}
         inputProps={isReactNative() ? inputProps : {}}
-        style={{ cursor: _isDisabled ? 'not-allowed' : 'pointer' }}
+        style={{
+          cursor: _isDisabled ? 'not-allowed' : 'pointer',
+          width: width as string | number,
+          maxWidth: maxWidth as string | number,
+          minWidth: minWidth as string | number,
+        }}
       >
-        <BaseBox display="flex" flexDirection="column">
+        <BaseBox
+          display="flex"
+          flexDirection="column"
+          width={width}
+          maxWidth={maxWidth}
+          minWidth={minWidth}
```

Comment:
Why are we passing width to multiple childs in this component? Shouldn't only parent component have width and child components take parent's width 🤔